libcontainer/integration/seccomp_test.go

```diff
@@ -77,10 +77,10 @@ func TestSeccompDenyGetcwdWithErrno(t *testing.T) {
 		t.Fatalf("Getcwd should fail with negative exit code, instead got %d!", exitCode)
 	}
 
-	expected := "pwd: getcwd: No such process"
+	expected := "getcwd: cannot access parent directories: No such process"
 	actual := strings.Trim(buffers.Stderr.String(), "\n")
-	if actual != expected {
-		t.Fatalf("Expected output %s but got %s\n", expected, actual)
+	if !strings.Contains(actual, expected) {
+		t.Fatalf("Expected output to contain `%s` but got `%s`\n", expected, actual)
```

Comment:
Can use `%q` here which does the quoting.

Reply:
Thanks addressed (and in other parts of the code)!